src/bootstrap/server-config.js

```diff
@@ -76,10 +76,12 @@ const configSchema = joi.object({
     identityProviders: joi.array().items(joi.object({
       key: joi.string().required(),
       displayName: joi.string().required(),
-      entryPoint: joi.string().required(),
-      cert: joi.string().required(),
       logoUrl: joi.string().allow(null).default(null),
-      expiryTimeoutInDays: joi.number().integer().min(1).default(6 * 30)
+      expiryTimeoutInDays: joi.number().integer().min(1).default(6 * 30),
+      metadata: joi.object({
+        url: joi.string().required(),
+        entityId: joi.string().required()
+      }).required()
```

Comment:
From setup perspective (as well) this is so much better/nicer! It's contained and simplified 😍

Reply:
Yeah, it makes it a lot easier.